docs/cookbooks/harness.rst

```diff
@@ -327,6 +327,24 @@ The mapping datafile must be modified to add the pool size.
 This will initialize 2 connections to the uut and the cli commands will go up
 to twice as fast! This number can scale up to what your device can support.
 
+In order to use connection pool for different connections, the new mapping schema needs to be followed.
```

Comment:
Do we support old and new schemas? How does the user know which schema is used? If there is only a single schema, we can drop the 'new mapping schema' nomenclature and just describe what is the required structure.

Reply:
yes @dwapstra we support both new and old schema.